test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java

```diff
@@ -430,6 +431,12 @@ public void TestQueryBatch(String inputFileName) throws SQLException, ClassNotFo
         File expectedFile;
         if (isParallelQueryMode && checkParallelQueryExpected)
             expectedFile = new File(parallelQueryGeneratedFilesDirectoryPath + outputFileName + ".out");
+        else if(serverCollationName != "default"){
```

Comment:
what happens when we run in parallel query mode with non-default server collation name?

Reply:
Execution remains same, if some query has different expected output in parallel query mode with non-default server collation compared to output in parallel query mode with default server collation then that test will fail with the corresponding output diff. Currently no support is added to provide different output in the above scenario.

Reply:
Added support to provide different output in the above scenario.